client/src/app/site/assignments/services/assignment-poll.service.ts

```diff
@@ -83,6 +88,7 @@ export class AssignmentPollService extends PollService {
                     if (Math.min(...yes) < 0) {
                         return null;
                     } else {
+                        // TODO: Counting 'No (and possibly 'Abstain') here?
```

Comment:
this is the discussion point I need: how do 'no to  all' and 'abstain' influence the percent base?

Reply:
in 2.3 client, these values don't affect the calculation. I'm unsure if they should.

Reply:
@emanuelschuetze Ping

Reply:
e clarified in direct discussion: We will continue as in 2.3